### PR TITLE
[BAD-597] Spoon: Logging during "Test" Hadoop Cluster does not contai…

### PR DESCRIPTION
**Pull request by stanislau-strelchanka**
[BAD-597] Spoon: Logging during "Test" Hadoop Cluster does not contain all exceptions. (MapR 5.2.0 client on Windows)

 - print stacktrace if test shim throws exception

### CODE REVIEW

**Comment by stanislau-strelchanka** [comment]
@hudak @mkambol @mbatchelor @mchen-len-son please review

File: impl/shim/shimTests/src/main/java/org/pentaho/big/data/impl/shim/tests/TestShimLoad.java [hunk 1]
```diff
@@ -75,11 +75,13 @@ public TestShimLoad( MessageGetterFactory messageGetterFactory,
           messageGetter.getMessage( TEST_SHIM_LOAD_SHIM_LOADED_MESSAGE, activeConfigurationId ),
           ClusterRuntimeTestEntry.DocAnchor.SHIM_LOAD ) );
     } catch ( NoShimSpecifiedException e ) {
+      e.printStackTrace();
```
Comment (VasilinaTerehova) [inline impl/shim/shimTests/src/main/java/org/pentaho/big/data/impl/shim/tests/TestShimLoad.java]: Why printstacktrace? Not logger using? 
This place logs all the exceptions at the end of tests  - https://github.com/pentaho/big-data-plugin/blob/6db1566b1843ddb6b1e57e9372ab84a7be8ad5d5/kettle-plugins/common/ui/src/main/java/org/pentaho/big/data/plugins/common/ui/ClusterTestDialog.java#L209. Does it print exception when detailed level set?

**Comment by stanislau-strelchanka** [comment]
@VasilinaTerehova please review

File: kettle-plugins/common/ui/src/test/java/org/pentaho/big/data/plugins/common/ui/TestClusterTestDialog.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    Mockito.when( runtimeTestModuleResults.getRuntimeTestResults() ).thenReturn( runtimeTestResults );
+    Mockito.when( result.getRuntimeTest() ).thenReturn( Mockito.mock( RuntimeTest.class ) );
+    Mockito.when( result.getRuntimeTestResultEntries() ).thenReturn( entries );
+    Mockito.when( entry.getSeverity() ).thenReturn( RuntimeTestEntrySeverity.FATAL );
+    Mockito.when( entry.getException() ).thenReturn( new Exception() );
+
+    testDialog.getRunnable( progressBar, clusterTestStatus, testLabel ).run();
+
+    // check that exception is called four times: 2 - for null check, 2 to print stack trace to log
+    Mockito.verify( entry, Mockito.times( 4 ) ).getException();
```
Comment (VasilinaTerehova) [inline kettle-plugins/common/ui/src/test/java/org/pentaho/big/data/plugins/common/ui/TestClusterTestDialog.java]: https://github.com/pentaho/pentaho-hadoop-shims/blob/master/common/hbase-comparators/src/test/java/org/pentaho/hbase/shim/common/CommonHBaseConnectionTest.java#L154
it looks better to mock LogChannel and check whether mock logBasic was called, not getException from entry

**Comment by stanislau-strelchanka** [comment]
@VasilinaTerehova 
updated tests and log in cluster test created using kettle log factory

**Comment by VasilinaTerehova** [comment]
thank you, now looks ok with log factory for me

**Comment by stanislau-strelchanka** [comment]
@VasilinaTerehova @hudak @mkambol @mbatchelor @mchen-len-son please review

**Pull request merged**